numpy/core/src/common/npy_cpu_dispatch.h

```diff
@@ -187,7 +187,7 @@
 /**
  * Macro NPY_CPU_DISPATCH_CALL(LEFT, ...) is used for runtime dispatching
  * of the exported functions and variables within the dispatch-able sources
- * according to the highested interesed CPU features that supported by the
+ * according to the highest interested CPU features that are supported by the
  * running machine depending on the required optimizations.
  *
```

Comment:
This paragraph could probably use a bit more work, "dispatchable" instead of "dispatch-able" and really, just a general rewrite for clarity. Not needed in the PR though.

Reply:
There are > 60 occurrences of `dispatch-able` in NumPy sources, and < 20 occurrences of `dispatch-able`. Do I change them all?

How about rewriting that praragraph as follows?
```
 * Macro NPY_CPU_DISPATCH_CALL(LEFT, ...) dispatches functions and
 * variables exported from dispatch-able sources, choosing the most
 * performant CPU features available on the current machine,
 * depending on the required optimizations.
```

Reply:
Dispatchable is the word that can be found in dictionaries, dispatch-able is understandable, but not a dictionary word.